src/styles/blocks/catalog.scss

```diff
@@ -17,6 +17,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform $time-duration, color $time-duration;
```

Comment:
the same, try to change order for transition - `color $time-duration,  transform $time-duration;`

<img width="415" alt="Screenshot 2022-12-14 at 17 21 28" src="https://user-images.githubusercontent.com/37452763/207635970-62a07f4c-60d5-46c0-b741-80dbed8fab22.png">